dist/main.js

```diff
@@ -0,0 +1,147 @@
+const renderer = new Renderer()
+const localStorageManager = new LocalStorageManager()
+
+const getRoster = function () {
+  const $input = $("#team-name-input")
+  if ($input.val().length) {
```

Comment:
extract the value to a variable, instead of invoking the `val()` function every time